### ISSUE TITLE
Subclass relation between Poison Artifact Function and Damaging Artifact Function might be mistaken

### ISSUE THREAD

**Issue by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#247]
According to the definition of Damaging Artifact Function, such a function must be realized in a process in which the structural integrity of an entity is impaired. Now I might be wrong here, but it would seem that some poisons do not act so as to impair the structural integrity of a living thing (at least given the definitions of "structural integrity" I've been able to find). Consider, for instance, a poison whose effect is to gradually slow a person's heart rate until it completely stops. Such a poison might leave the person's structure entirely unaffected.

Of course, depending on the intent behind the Poison Artifact Function class, this might not be a problem. For instance, if only some poisons (i.e., those that do impair structural integrity) are intended to have a Poison Artifact Function, then everything's fine. However, if all poisons are intended to have a Poison Artifact Function, then (assuming the considerations of the previous paragraph are correct) Poison Artifact Function ought no be a subclass of Damaging Artifact Function.

**Comment by cameronmore** [comment]
This definitely needs to be fixed, I'm just thinking of whether there's a good parent class for both damaging and poisoning or we expand damaging.

I would think that damaging could definitely include poisoning. I wouldn't want to get into harming, that has too much philosophical baggage and isn't what we mean.

**Comment by cameronmore** [comment]
Damaging = physical and/or structural impairment to a material entity which may cause a loss or decrease of disposition of that material entity.

Does 'physical' go far enough? Or adding 'may cause a loss or decrease of disposition' cover it?

Just throwing out ideas.

**Comment by gregfowlerphd** [comment]
I think something like your proposal might be on the right track, @cameronmore.

**Comment by cameronmore** [comment]
Thinking about this more, I want to coordinate this term with Damaged Stasis:

`Dameged Stasis = A Stasis of Specifically Dependent Continuant in which some Independent Continuant bears a Quality or Realizable Entity that has suffered impairment (i.e., a decrease or loss) due to a previous action or event such that the Independent Continuant is now of lesser value, usefulness, or functionality.`

So, 'impairment' is such that an Independent Continuant is "of lesser value, usefulness, or functionality." We can expand the current definition of Damaging Artifact Function reflecting this:

`Damaging Artifact Function = An Artifact Function that is realized in processes in which its bearer is used to cause structural impairment, deterioration, lessening or ceasing of a Disposition of some Independent Continuant.` (welcome to wordsmithing of course)

(As a reminder, the current definition is `An Artifact Function that is realized in a process in which the structural integrity of an entity is impaired.`)

**Comment by gregfowlerphd** [comment]
@cameronmore: I'm not opposed to your proposal to coordinate Damaging Artifact Function with Damaged Stasis, and I'll continue to think about it. However, I think that your proposed definition for Damaging Artifact Function, as it currently stands, is problematic: Suppose an artifact's function is to make independent continuants of a certain sort less fragile. Then the definition implies that function is a damaging artifact function, which seems incorrect.

**Comment by cameronmore** [comment]
That's a good point. We could just absorb the phrase 'lesser value, usefulness, or functionality' but I think more precision would be nice, maybe with a mention 'loss of function' or loss of a disposition in a purely negative sense.

**Comment by gregfowlerphd** [comment]
@cameronmore: I think your 'loss of function' suggestion might work. Are you suggesting something like this?:

> **Damaging Artifact Function:** An Artifact Function that is realized in processes in which its bearer is used to cause structural impairment, deterioration, or the lessening or ceasing of a Function of some Independent Continuant.

**Comment by cameronmore** [comment]
Yeah I agree, but want to include disposition for cases like if a cave if being damaged by a piece of mining equipment

**Comment by gregfowlerphd** [comment]
Ah. Do you think we can include dispositions without running afoul of the fragility problem I mentioned above?

**Comment by cameronmore** [comment]
Thinking about it more, I think this is fine:

Damaging Artifact Function: An Artifact Function that is realized in processes in which its bearer is used to cause structural impairment, deterioration, or the lessening or ceasing of the value, Qualities, or Dispositions of an Independent Continuant.

And adding an elucidation or comment to the effect that this class is not intended to cover cases where an artifact is used to get rid of an Independent Continuant's dispositions like fragility or the softness of unfired clay?

I will continue to think about a better definition, but I am resisting adding normative language about the fact that we associate damage with 'negative effects' (hoping that "the lessening or ceasing of the value" covers that implication)

Thoughts?

**Comment by gregfowlerphd** [comment]
@cameronmore: I'm willing to defer to you regarding your proposal. I admit to some unease about providing a definition but also adding a comment (or elucidation) that says, essentially, that the definition is incorrect. However, you have more experience here, putting you in a better position to judge whether it's appropriate to do so.

That said, I've realized something that might be worth noting: The "dispositions issue" with your proposed definition of DAF isn't limited to somewhat odd cases like those you mentioned. It also affects cases in which the notion of a disposition has been used in ontological modeling. For example, suppose an artifact's function is to cure a disease. Then given the definition of 'disease' in OGMS, your proposed definition would seem to have the consequence that that function is a DAF.

**Comment by cameronmore** [comment]
Thank you for your deferral, though I'll keep thinking about it because that is a good case regarding the definition of disease. Maybe we can connect offline

**Comment by cameronmore** [comment]
As we discussed offline, the definition could use improvement but we do not have a concrete proposal, so it makes sense to convert this to a discussion.
